velox/functions/sparksql/tests/FromJsonTest.cpp

```diff
@@ -167,6 +167,35 @@ TEST_F(FromJsonTest, basicDouble) {
   testFromJson(input, makeRowVector({"a"}, {expected}));
 }
 
+TEST_F(FromJsonTest, basicDate) {
+  auto expected = makeNullableFlatVector<int32_t>(
+      {18809,
+       18809,
+       18809,
+       0,
+       18809,
+       -713975,
+       15,
+       std::nullopt,
+       std::nullopt,
+       std::nullopt,
+       std::nullopt},
+      DATE());
+  auto input = makeFlatVector<std::string>(
+      {R"({"a": "2021-07-01T"})",
+       R"({"a": "2021-07-01"})",
+       R"({"a": "2021-7"})",
+       R"({"a": "1970"})",
+       R"({"a": "2021-07-01GMT"})",
```

Comment:
Is the GMT only be the end? If that, we can use StringView(dateStr.data(), size - gmt_size)

Reply:
Done, thanks